CalPatRec/src/PhiClusterFinder_module.cc

```diff
@@ -0,0 +1,566 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+///////////////////////////////////////////////////////////////////////////////
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Principal/Handle.h"
+#include "art/Framework/Core/EDProducer.h"
+#include "art_root_io/TFileService.h"
+#include "art/Utilities/make_tool.h"
+
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Sequence.h"
+#include "fhiclcpp/ParameterSet.h"
+#include "CLHEP/Units/PhysicalConstants.h"
+
+#include "Offline/GeometryService/inc/GeometryService.hh"
+#include "Offline/GeometryService/inc/GeomHandle.hh"
+#include "Offline/RecoDataProducts/inc/StrawHitPosition.hh"
+#include "Offline/RecoDataProducts/inc/TimeCluster.hh"
+#include "Offline/RecoDataProducts/inc/CaloCluster.hh"
+#include "Offline/RecoDataProducts/inc/ComboHit.hh"
+#include "Offline/RecoDataProducts/inc/StrawHit.hh"
+
+#include "Offline/TrkReco/inc/TrkUtilities.hh"
+#include "Offline/TrkReco/inc/TrkTimeCalculator.hh"
+#include "Offline/GeneralUtilities/inc/Angles.hh"
+#include "Offline/ConfigTools/inc/ConfigFileLookupPolicy.hh"
+#include "Offline/Mu2eUtilities/inc/ModuleHistToolBase.hh"
+#include "Offline/Mu2eUtilities/inc/polyAtan2.hh"
+#include "Offline/CalPatRec/inc/PhiClusterFinder_types.hh"
+
+#include "TH1.h"
+#include "TF1.h"
+#include "TROOT.h"
+#include "TMath.h"
+
+#include <boost/accumulators/accumulators.hpp>
+#include <boost/accumulators/statistics/weighted_median.hpp>
+#include <boost/accumulators/statistics/stats.hpp>
+#include <boost/accumulators/statistics/mean.hpp>
+#include <boost/accumulators/statistics/max.hpp>
+#include <boost/accumulators/statistics/min.hpp>
+#include <boost/accumulators/statistics/weighted_mean.hpp>
+
+#include <iostream>
+#include <fstream>
+#include <memory>
+#include <algorithm>
+#include <utility>
+#include <functional>
+#include <vector>
+
+using namespace std;
+using namespace boost::accumulators;
```

Comment:
the same applies here. The risk is that a user may use a variable that has the same name as one of the functions defined in these libraries and in some rare cases that can create problems because the compiler doesn't spot it.